pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -120,12 +120,8 @@ in stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
-  # Required for $SQLCIPHER_LIB which contains "/build/" inside the path:
-  noAuditTmpdir = true;
-
   preFixup = ''
-    export SQLCIPHER_LIB="$out/lib/Signal/resources/app.asar.unpacked/node_modules/better-sqlite3/build/Release/better_sqlite3.node"
-    test -x "$SQLCIPHER_LIB" # To ensure the location hasn't changed
+    export SQLCIPHER_LIB="${sqlcipher}/lib/libsqlcipher.so"
```

Comment:
We can also drop `$SQLCIPHER_LIB` then (was only a dirty "hack" for the test).

Reply:
Done.